MultiLangProgGuide/MultiLangProgGuide.rst

```diff
@@ -1819,6 +1821,920 @@ If the requested :code:`timeout_nsec` exceeds
 :code:`YDB_MAX_TIME_NSEC`, the function returns
 :code:`YDB_ERR_TIME2LONG`; otherwise it returns :code:`YDB_OK`.
 
+=================
+Programming in Go
+=================
+
+Programming YottaDB in the `Go language <https://golang.org/>`_ is
+accomplished through a wrapper that uses `cgo
+<https://golang.org/cmd/cgo/>`_ to provide the Go programmer with a
+YottaDB package. YottaDB C functions are wrapped by Go methods where a
+method can meaningfully be associated with a Go structure, and Go
+functions otherwise.
+
+As the Go language has important differences from C (for example it
+lacks macros, but has methods), below are Go-specific sections of the
+`Quick Start`_, `Concepts`_, `Symbolic Constants`_, `Data Structures &
+Type Definitions`_ and `Simple API`_ sections above. The sections
+below specific to Go are intended to supplement their general (C)
```

Comment:
"The sections specific to Go below" or "The sections below that are specific to Go"

Reply:
Done